pages/2017/index.rst

```diff
@@ -8,4 +8,53 @@ Python in Astronomy 2017
 Lorentz Center, Leiden
 ----------------------
 
-WOOOO YEAG
+The third edition of the Python in Astronomy Conference will be held at the
+Lorentz Center on the **8 - 12th May 2017**.
+
+As with the two previous Python in Astronomy conferences we aim to bring a wide
+variety of people who use, develop or teach people about Python packages in the
+context of all forms of Astronomy. The conference will include presentations,
+tutorials unconference sessions and sprints. As well as building the community
+around astronomical uses of Python the conference aims to improve collaboration
+and interoperability between Python packages, share knowledge on Python packages
+and techniques and provide training and educational materials for users and
+developers of Python packages.
+
+Attending PyAstro17
+###################
+
+Applications for Python in Astronomy 2017 will open on the September 12th 2016,
+participants will be selected both on merit and to ensure a diversity at the
+conference.
+
```

Comment:
agreed. How about “people who are new to contributing to open source
software”?
How about also adding something explicit like this:
Only basic familiarity with Python is expected of participants. This
conference is neither intended to be a introduction to Python bootcamp nor
only for expert-level Python developers.

On Wed, Jul 27, 2016 at 12:46 PM, Matt Craig <notifications@github.com>
wrote:

> In pages/2017/index.rst
> <https://github.com/OpenAstronomy/pyastro/pull/1#discussion_r72476464>:
>
> > +As with the two previous Python in Astronomy conferences we aim to bring a wide
> > +variety of people who use, develop or teach people about Python packages in the
> > +context of all forms of Astronomy. The conference will include presentations,
> > +tutorials unconference sessions and sprints. As well as building the community
> > +around astronomical uses of Python the conference aims to improve collaboration
> > +and interoperability between Python packages, share knowledge on Python packages
> > +and techniques and provide training and educational materials for users and
> > +developers of Python packages.
> > +
> > +Attending PyAstro17
> > +###################
> > +
> > +Applications for Python in Astronomy 2017 will open on the September 12th 2016,
> > +participants will be selected both on merit and to ensure a diversity at the
> > +conference.
> > +
>
> Participant selection will be made with the goal of *growing* the Python
> in Astronomy community
>
> Agreed; we should talk through how many to set allocate for new vs
> returning members.
>
> junior astronomers and people who are new to Python.
>
> What about "junior astronomers and users of python software"? I do think
> we want to avoid the word "new" (though at some point a separate meeting to
> do that might be useful).
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenAstronomy/pyastro/pull/1/files/b7e164dc00d89f8f56fef20e4ca6fa5abc35d7fd#r72476464>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABYJmcdU2_uDieSvD68wBxw5KrC8Rq5yks5qZ4tKgaJpZM4JU92Q>
> .
>



-- 
Kelle Cruz, PhD — http://kellecruz.com/
917.725.1334 — Hunter: x16486 — AMNH: x3404